doc/README.md

```diff
@@ -16,7 +16,7 @@ pip install -r requirements-doc.txt
 To compile the documentation and open it locally, run the following command from this directory.
 
 ```bash
-make html && open _build/html/index.html
+make develop && open _build/html/index.html
```

Comment:
Can you also make sure to explicitly say that this is for development, and to reproduce build failures you should still run make html?

Reply:
added